runtime/Cpp/runtime/src/tree/xpath/XPathLexer.cpp

```diff
@@ -46,14 +47,14 @@ const atn::ATN& XPathLexer::getATN() const {
 
 void XPathLexer::action(RuleContext *context, size_t ruleIndex, size_t actionIndex) {
   switch (ruleIndex) {
-    case 4: IDAction(dynamic_cast<RuleContext *>(context), actionIndex); break;
+    case 4: IDAction(dynamic_cast<antlr4::RuleContext *>(context), actionIndex); break;
 
   default:
     break;
   }
 }
 
-void XPathLexer::IDAction(RuleContext * /*context*/, size_t actionIndex) {
+void XPathLexer::IDAction(antlr4::RuleContext *context, size_t actionIndex) {
```

Comment:
These files generated with the 4.6 version of ANTLR. On my opinion, ALL generated files should be excluded from the repository.